### DISCUSSION TITLE
Git issue on ssh private keys

### DISCUSSION THREAD

**Discussion by johndave9296** [requarks/wiki#6799] [Help / Questions] [answered]
I'm currently experiencing this error and still tracing some fixes and issue on this. Hopefully someone can help me with it.

`Fetching origin Load key "/home/johndave-me/wiki/data/secure/git-ssh.pem": error in libcrypto git@github.com: Permission denied (publickey). fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists. error: Could not fetch origin`

**Comment by NGPixel** [comment, marked as answer]
Either your SSH key is wrong or you don't have the permissions set on your repo for that key.

**Reply by johndave9296** [reply]
Just figured it out. Used a multiple ssh via config file and setting permissions to be **777** on git
